cogboard-webapp/src/components/widgets/types/PersonDrawWidget/styled.js

```diff
@@ -2,6 +2,6 @@ import styled from '@emotion/styled/macro';
 
 import { Typography } from '@material-ui/core';
 
-export const StyledTypography = styled(Typography)`
-  height: 100%;
+export const CenterHorizontal = styled(Typography)`
```

Comment:
For me better name would be `CenteredTypography`

Reply:
Fixed